(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 100k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 100k /dev/urandom > /tmp/tmpsaky1efs
Data Size: 100k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpsaky1efs  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
1.325995	0.000000	1.326055		2.107262	0		200		102400		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 200k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 200k /dev/urandom > /tmp/tmptwy9mqra
Data Size: 200k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmptwy9mqra  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319042	0.000000	0.319101		1.701617	0		200		204800		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 300k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 300k /dev/urandom > /tmp/tmpui3iqtlg
Data Size: 300k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpui3iqtlg  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319286	0.000000	0.319345		4.376392	0		200		307200		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 400k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 400k /dev/urandom > /tmp/tmprl_5umrr
Data Size: 400k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmprl_5umrr  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324329	0.000000	0.324386		5.055604	0		200		409600		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 500k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 500k /dev/urandom > /tmp/tmpncvqnz5o
Data Size: 500k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpncvqnz5o  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319014	0.000000	0.319072		2.387647	0		200		512000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 600k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 600k /dev/urandom > /tmp/tmplzljc0b9
Data Size: 600k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmplzljc0b9  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330109	0.000000	0.330166		2.157022	0		200		614400		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 700k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 700k /dev/urandom > /tmp/tmph5qe8ev4
Data Size: 700k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmph5qe8ev4  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319041	0.000000	0.319097		6.293442	0		200		716800		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 800k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 800k /dev/urandom > /tmp/tmpyk98q4x2
Data Size: 800k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpyk98q4x2  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319046	0.000000	0.319105		4.445145	0		200		819200		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 900k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 900k /dev/urandom > /tmp/tmpwiwjswwn
Data Size: 900k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpwiwjswwn  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319064	0.000000	0.319123		3.022794	0		200		921600		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 1000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 1000k /dev/urandom > /tmp/tmpo7taj61j
Data Size: 1000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpo7taj61j  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.325369	0.000000	0.325426		3.105081	0		200		1024000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 2000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 2000k /dev/urandom > /tmp/tmp3dhmf2aw
Data Size: 2000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp3dhmf2aw  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 60 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318916	0.000000	0.318974		6.824885	0		200		2048000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 3000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 3000k /dev/urandom > /tmp/tmp7lhgxgg0
Data Size: 3000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp7lhgxgg0  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 90 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318998	0.000000	0.319062		10.385905	0		200		3072000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 4000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 4000k /dev/urandom > /tmp/tmp9kdpxtm5
Data Size: 4000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp9kdpxtm5  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 120 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318826	0.000000	0.318883		8.659401	0		200		4096000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 5000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 5000k /dev/urandom > /tmp/tmpdk1jlxw1
Data Size: 5000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpdk1jlxw1  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 150 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330272	0.000000	0.330329		12.789280	0		200		5120000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 6000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 6000k /dev/urandom > /tmp/tmps6cucm0s
Data Size: 6000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmps6cucm0s  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 180 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319047	0.000000	0.319105		15.249171	0		200		6144000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 7000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 7000k /dev/urandom > /tmp/tmpplroy4j3
Data Size: 7000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpplroy4j3  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 210 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319156	0.000000	0.319212		14.457412	0		200		7168000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 8000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 8000k /dev/urandom > /tmp/tmptlblld90
Data Size: 8000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmptlblld90  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 240 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319020	0.000000	0.319077		20.894042	0		200		8192000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 9000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 9000k /dev/urandom > /tmp/tmpjohii506
Data Size: 9000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpjohii506  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 270 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.326767	0.000000	0.326824		18.764514	0		200		9216000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 10000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 10000k /dev/urandom > /tmp/tmpy63j9i0g
Data Size: 10000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpy63j9i0g  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 300 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324920	0.000000	0.324978		21.172486	0		200		10240000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 11000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 11000k /dev/urandom > /tmp/tmpa2wjxn13
Data Size: 11000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpa2wjxn13  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 330 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
1.352363	0.000000	1.352421		22.743625	0		200		11264000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 12000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 12000k /dev/urandom > /tmp/tmpds6dd4iv
Data Size: 12000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpds6dd4iv  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 360 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319034	0.000000	0.319094		27.856149	0		200		12288000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 13000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 13000k /dev/urandom > /tmp/tmpqdtii11s
Data Size: 13000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpqdtii11s  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 390 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319164	0.000000	0.319221		26.731661	0		200		13312000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 14000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 14000k /dev/urandom > /tmp/tmp87h_tmnn
Data Size: 14000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp87h_tmnn  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 420 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.325382	0.000000	0.325439		31.022935	0		200		14336000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 15000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 15000k /dev/urandom > /tmp/tmpos8i5ksw
Data Size: 15000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpos8i5ksw  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 450 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.325023	0.000000	0.325079		30.384397	0		200		15360000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 16000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 16000k /dev/urandom > /tmp/tmp3c0puxxi
Data Size: 16000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp3c0puxxi  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 480 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324299	0.000000	0.324358		36.729738	0		200		16384000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 17000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 17000k /dev/urandom > /tmp/tmpltee827a
Data Size: 17000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpltee827a  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 510 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330206	0.000000	0.330264		34.881105	0		200		17408000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 18000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 18000k /dev/urandom > /tmp/tmpmxnaqh4j
Data Size: 18000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpmxnaqh4j  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 540 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330468	0.000000	0.330525		38.017320	0		200		18432000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 19000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 19000k /dev/urandom > /tmp/tmpg482ju1i
Data Size: 19000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpg482ju1i  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 570 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318984	0.000000	0.319040		40.504410	0		200		19456000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 20000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 20000k /dev/urandom > /tmp/tmpsm18h1cz
Data Size: 20000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpsm18h1cz  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 600 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330533	0.000000	0.330592		44.399048	0		200		20480000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 30000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 30000k /dev/urandom > /tmp/tmpjlmqwyx6
Data Size: 30000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpjlmqwyx6  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 900 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319020	0.000000	0.319076		61.251125	0		200		30720000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 40000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 40000k /dev/urandom > /tmp/tmpv0qgs3na
Data Size: 40000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpv0qgs3na  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 1200 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330364	0.000000	0.330421		83.203642	0		200		40960000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 50000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 50000k /dev/urandom > /tmp/tmpumflyoh3
Data Size: 50000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpumflyoh3  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 1500 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318986	0.000000	0.319043		102.199409	0		200		51200000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 60000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 60000k /dev/urandom > /tmp/tmpd1b1hsxy
Data Size: 60000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpd1b1hsxy  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 1800 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330436	0.000000	0.330493		115.930060	0		200		61440000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 70000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 70000k /dev/urandom > /tmp/tmpjd_8xumj
Data Size: 70000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpjd_8xumj  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 2100 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330440	0.000000	0.330495		146.429723	0		200		71680000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 80000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 80000k /dev/urandom > /tmp/tmpmnj6nbxm
Data Size: 80000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpmnj6nbxm  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 2400 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330352	0.000000	0.330406		162.751066	0		200		81920000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 90000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 90000k /dev/urandom > /tmp/tmp7jhqwmym
Data Size: 90000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp7jhqwmym  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 2700 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330373	0.000000	0.330454		183.745492	0		200		92160000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 100k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 100k /dev/urandom > /tmp/tmpkfxvfk0e
Data Size: 100k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpkfxvfk0e  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324058	0.000000	0.324113		1.588077	0		200		102400		18	
0.164231	0.000000	0.164287		0.925768	0		200		102400		18	
0.164204	0.000000	0.164259		0.780939	0		200		102400		18	
0.164149	0.000000	0.164205		1.471162	0		200		102400		18	
0.158448	0.000000	0.158506		1.474279	0		200		102400		18	
0.158336	0.000000	0.158391		1.460426	0		200		102400		18	
0.158342	0.000000	0.158397		0.791714	0		200		102400		18	
0.158228	0.000000	0.158283		1.464646	0		200		102400		18	
0.158565	0.000000	0.158619		1.471188	0		200		102400		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 200k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 200k /dev/urandom > /tmp/tmp8z8ilsts
Data Size: 200k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp8z8ilsts  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.707075	0.000000	0.707134		1.797352	0		200		204800		18	
0.158517	0.000000	0.158572		1.688341	0		200		204800		18	
0.158330	0.000000	0.158384		1.605714	0		200		204800		18	
0.158404	0.000000	0.158460		1.081377	0		200		204800		18	
0.158350	0.000000	0.158405		1.109043	0		200		204800		18	
0.164177	0.000000	0.164231		1.615913	0		200		204800		18	
0.164196	0.000000	0.164254		1.093872	0		200		204800		18	
0.158532	0.000000	0.158587		1.092264	0		200		204800		18	
0.158590	0.000000	0.158648		1.580880	0		200		204800		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 300k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 300k /dev/urandom > /tmp/tmp60hq2jb7
Data Size: 300k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp60hq2jb7  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324138	0.000000	0.324192		1.940994	0		200		307200		18	
0.164907	0.000000	0.165002		2.289996	0		200		307200		18	
0.164183	0.000000	0.164239		1.736150	0		200		307200		18	
0.164211	0.000000	0.164266		1.820101	0		200		307200		18	
0.164189	0.000000	0.164244		1.257540	0		200		307200		18	
0.164176	0.000000	0.164233		1.833443	0		200		307200		18	
0.158329	0.000000	0.158384		2.264052	0		200		307200		18	
0.158392	0.000000	0.158446		2.158558	0		200		307200		18	
0.158408	0.000000	0.158481		2.483196	0		200		307200		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 400k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 400k /dev/urandom > /tmp/tmp35fasorb
Data Size: 400k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp35fasorb  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318912	0.000000	0.318965		2.260369	0		200		409600		18	
0.158379	0.000000	0.158435		2.625760	0		200		409600		18	
0.158382	0.000000	0.158436		1.289875	0		200		409600		18	
0.158255	0.000000	0.158310		2.936525	0		200		409600		18	
0.158286	0.000000	0.158341		1.743750	0		200		409600		18	
0.158362	0.000000	0.158419		1.830386	0		200		409600		18	
0.170741	0.000000	0.170841		2.513704	0		200		409600		18	
0.164171	0.000000	0.164226		2.273781	0		200		409600		18	
0.164165	0.000000	0.164221		2.043164	0		200		409600		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 500k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 500k /dev/urandom > /tmp/tmput50fbal
Data Size: 500k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmput50fbal  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330208	0.000000	0.330264		3.398968	0		200		512000		18	
0.164243	0.000000	0.164298		2.094616	0		200		512000		18	
0.164153	0.000000	0.164217		2.405033	0		200		512000		18	
0.158354	0.000000	0.158411		2.184601	0		200		512000		18	
0.158348	0.000000	0.158404		1.989755	0		200		512000		18	
0.158572	0.000000	0.158626		3.380880	0		200		512000		18	
0.164100	0.000000	0.164154		2.172937	0		200		512000		18	
0.158615	0.000000	0.158672		2.065976	0		200		512000		18	
0.158353	0.000000	0.158408		2.197001	0		200		512000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 600k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 600k /dev/urandom > /tmp/tmpdbs5j1p3
Data Size: 600k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpdbs5j1p3  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.323794	0.000000	0.323849		4.462449	0		200		614400		18	
0.158258	0.000000	0.158313		1.844449	0		200		614400		18	
0.158312	0.000000	0.158367		2.727702	0		200		614400		18	
0.164248	0.000000	0.164304		2.270988	0		200		614400		18	
0.164066	0.000000	0.164121		2.326542	0		200		614400		18	
0.158423	0.000000	0.158478		2.767429	0		200		614400		18	
0.158298	0.000000	0.158354		3.695761	0		200		614400		18	
0.158306	0.000000	0.158361		2.187692	0		200		614400		18	
0.158325	0.000000	0.158380		2.192807	0		200		614400		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 700k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 700k /dev/urandom > /tmp/tmpzh4ltt4i
Data Size: 700k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpzh4ltt4i  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.323921	0.000000	0.323978		29.999015	28		000		716800		0	Operation timed out after 29999 milliseconds with 0 bytes received
0.158347	0.000000	0.158402		3.066959	0		200		716800		18	
0.158275	0.000000	0.158332		2.521461	0		200		716800		18	
0.158555	0.000000	0.158611		2.289980	0		200		716800		18	
0.158316	0.000000	0.158374		2.784095	0		200		716800		18	
0.158226	0.000000	0.158279		2.408831	0		200		716800		18	
0.158358	0.000000	0.158413		2.731172	0		200		716800		18	
0.158595	0.000000	0.158651		2.795254	0		200		716800		18	
0.164022	0.000000	0.164079		3.250660	0		200		716800		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 800k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 800k /dev/urandom > /tmp/tmpkksdu7jx
Data Size: 800k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpkksdu7jx  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324697	0.000000	0.324752		3.232996	0		200		819200		18	
0.158225	0.000000	0.158279		2.177206	0		200		819200		18	
0.158327	0.000000	0.158380		2.847813	0		200		819200		18	
0.164162	0.000000	0.164217		2.259392	0		200		819200		18	
0.158481	0.000000	0.158537		3.856583	0		200		819200		18	
0.164102	0.000000	0.164158		5.205104	0		200		819200		18	
0.164143	0.000000	0.164199		2.485286	0		200		819200		18	
0.158377	0.000000	0.158434		2.730582	0		200		819200		18	
0.158368	0.000000	0.158425		2.555596	0		200		819200		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 900k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 900k /dev/urandom > /tmp/tmpisyktmzg
Data Size: 900k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpisyktmzg  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330388	0.000000	0.330443		3.479600	0		200		921600		18	
0.164141	0.000000	0.164195		2.556803	0		200		921600		18	
0.158298	0.000000	0.158353		5.334670	0		200		921600		18	
0.158281	0.000000	0.158335		2.973556	0		200		921600		18	
0.158508	0.000000	0.158561		2.670619	0		200		921600		18	
0.158509	0.000000	0.158565		2.756278	0		200		921600		18	
0.158285	0.000000	0.158340		7.014245	0		200		921600		18	
0.158476	0.000000	0.158533		5.517112	0		200		921600		18	
0.158547	0.000000	0.158603		2.320447	0		200		921600		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 1000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 1000k /dev/urandom > /tmp/tmposqoovz5
Data Size: 1000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmposqoovz5  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318868	0.000000	0.318940		4.908709	0		200		1024000		18	
0.164143	0.000000	0.164202		2.782959	0		200		1024000		18	
0.158283	0.000000	0.158337		2.651583	0		200		1024000		18	
0.158341	0.000000	0.158397		2.977482	0		200		1024000		18	
0.158310	0.000000	0.158367		4.169387	0		200		1024000		18	
0.158232	0.000000	0.158290		2.757445	0		200		1024000		18	
0.158328	0.000000	0.158382		2.659290	0		200		1024000		18	
0.158289	0.000000	0.158347		4.478687	0		200		1024000		18	
0.158220	0.000000	0.158276		5.665354	0		200		1024000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 2000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 2000k /dev/urandom > /tmp/tmpngsl5e_e
Data Size: 2000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpngsl5e_e  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 60 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
1.340044	0.000000	1.340099		5.459944	0		200		2048000		18	
0.158529	0.000000	0.158584		8.720570	0		200		2048000		18	
0.158270	0.000000	0.158325		5.366079	0		200		2048000		18	
0.158284	0.000000	0.158340		5.381969	0		200		2048000		18	
0.158319	0.000000	0.158375		6.805010	0		200		2048000		18	
0.158329	0.000000	0.158386		5.151376	0		200		2048000		18	
0.158300	0.000000	0.158356		5.629085	0		200		2048000		18	
0.164114	0.000000	0.164173		4.858863	0		200		2048000		18	
0.158263	0.000000	0.158320		4.314537	0		200		2048000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 3000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 3000k /dev/urandom > /tmp/tmpqg1q7g1r
Data Size: 3000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpqg1q7g1r  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 90 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324470	0.000000	0.324528		8.024053	0		200		3072000		18	
0.158343	0.000000	0.158408		7.246973	0		200		3072000		18	
0.158381	0.000000	0.158438		10.032926	0		200		3072000		18	
0.158313	0.000000	0.158368		9.083999	0		200		3072000		18	
0.158278	0.000000	0.158333		89.999014	28		000		3072000		0	Operation timed out after 89999 milliseconds with 0 bytes received
0.164218	0.000000	0.164276		8.581749	0		200		3072000		18	
0.158307	0.000000	0.158362		10.418107	0		200		3072000		18	
0.164100	0.000000	0.164157		9.845719	0		200		3072000		18	
0.164225	0.000000	0.164282		12.124198	0		200		3072000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 4000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 4000k /dev/urandom > /tmp/tmpju6tug2c
Data Size: 4000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpju6tug2c  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 120 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319399	0.000000	0.319455		9.836938	0		200		4096000		18	
0.158308	0.000000	0.158366		9.377030	0		200		4096000		18	
0.158720	0.000000	0.158777		10.446398	0		200		4096000		18	
0.158443	0.000000	0.158497		10.482320	0		200		4096000		18	
0.164102	0.000000	0.164158		10.170422	0		200		4096000		18	
0.158423	0.000000	0.158479		9.291869	0		200		4096000		18	
0.158302	0.000000	0.158359		9.065625	0		200		4096000		18	
0.164057	0.000000	0.164114		13.355639	0		200		4096000		18	
0.158313	0.000000	0.158370		11.918406	0		200		4096000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 5000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 5000k /dev/urandom > /tmp/tmpia3u3uma
Data Size: 5000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpia3u3uma  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 150 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
1.327173	0.000000	1.327229		15.484757	0		200		5120000		18	
0.158518	0.000000	0.158574		11.241660	0		200		5120000		18	
0.158302	0.000000	0.158358		11.497863	0		200		5120000		18	
0.158290	0.000000	0.158345		10.507786	0		200		5120000		18	
0.158301	0.000000	0.158355		14.531131	0		200		5120000		18	
0.158352	0.000000	0.158409		12.603989	0		200		5120000		18	
0.164196	0.000000	0.164251		12.139974	0		200		5120000		18	
0.164103	0.000000	0.164158		9.993206	0		200		5120000		18	
0.158232	0.000000	0.158288		12.247282	0		200		5120000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 6000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 6000k /dev/urandom > /tmp/tmputkwraat
Data Size: 6000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmputkwraat  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 180 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318937	0.000000	0.318993		13.450292	0		200		6144000		18	
0.158321	0.000000	0.158376		14.658811	0		200		6144000		18	
0.158331	0.000000	0.158387		14.744767	0		200		6144000		18	
0.158384	0.000000	0.158440		15.440024	0		200		6144000		18	
0.158304	0.000000	0.158359		13.394710	0		200		6144000		18	
0.158331	0.000000	0.158396		14.262695	0		200		6144000		18	
0.158329	0.000000	0.158383		12.720283	0		200		6144000		18	
0.164138	0.000000	0.164194		18.964590	0		200		6144000		18	
0.164142	0.000000	0.164198		11.575031	0		200		6144000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 7000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 7000k /dev/urandom > /tmp/tmpa35cr3r2
Data Size: 7000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpa35cr3r2  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 210 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319012	0.000000	0.319071		15.001672	0		200		7168000		18	
0.164057	0.000000	0.164114		19.194100	0		200		7168000		18	
0.158293	0.000000	0.158348		13.968474	0		200		7168000		18	
0.158513	0.000000	0.158570		14.634793	0		200		7168000		18	
0.158545	0.000000	0.158600		17.630521	0		200		7168000		18	
0.158292	0.000000	0.158348		18.138363	0		200		7168000		18	
0.164049	0.000000	0.164103		16.759475	0		200		7168000		18	
0.158386	0.000000	0.158440		16.896108	0		200		7168000		18	
0.158417	0.000000	0.158474		15.113261	0		200		7168000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 8000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 8000k /dev/urandom > /tmp/tmp86zq5koh
Data Size: 8000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp86zq5koh  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 240 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319248	0.000000	0.319303		17.171249	0		200		8192000		18	
0.158303	0.000000	0.158362		17.065745	0		200		8192000		18	
0.158585	0.000000	0.158641		17.846949	0		200		8192000		18	
0.158300	0.000000	0.158357		21.131136	0		200		8192000		18	
0.158309	0.000000	0.158374		19.379908	0		200		8192000		18	
0.164242	0.000000	0.164299		19.177270	0		200		8192000		18	
0.163443	0.000000	0.163498		17.944419	0		200		8192000		18	
0.158313	0.000000	0.158367		20.963705	0		200		8192000		18	
0.164104	0.000000	0.164160		17.947335	0		200		8192000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 9000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 9000k /dev/urandom > /tmp/tmpnbzjaazp
Data Size: 9000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpnbzjaazp  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 270 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324486	0.000000	0.324540		20.306181	0		200		9216000		18	
0.164184	0.000000	0.164239		18.599453	0		200		9216000		18	
0.158271	0.000000	0.158326		19.254037	0		200		9216000		18	
0.158311	0.000000	0.158365		18.793471	0		200		9216000		18	
0.158288	0.000000	0.158344		17.080864	0		200		9216000		18	
1.465769	0.000000	1.465829		22.059535	0		200		9216000		18	
0.164115	0.000000	0.164170		19.116539	0		200		9216000		18	
0.164234	0.000000	0.164290		20.899343	0		200		9216000		18	
0.164122	0.000000	0.164177		20.058455	0		200		9216000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 10000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 10000k /dev/urandom > /tmp/tmpf6jmqng3
Data Size: 10000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpf6jmqng3  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 300 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330009	0.000000	0.330064		20.226741	0		200		10240000		18	
0.158487	0.000000	0.158543		22.692201	0		200		10240000		18	
0.164103	0.000000	0.164160		21.537793	0		200		10240000		18	
0.164086	0.000000	0.164143		21.227251	0		200		10240000		18	
0.164084	0.000000	0.164140		20.342209	0		200		10240000		18	
0.158308	0.000000	0.158364		21.667671	0		200		10240000		18	
0.158262	0.000000	0.158317		22.279018	0		200		10240000		18	
0.164026	0.000000	0.164085		22.290700	0		200		10240000		18	
0.164224	0.000000	0.164282		22.297625	0		200		10240000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 700k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 700k /dev/urandom > /tmp/tmpmitq7k6z
Data Size: 700k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpmitq7k6z  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324895	0.000000	0.324951		29.999014	28		000		716800		0	Operation timed out after 29999 milliseconds with 0 bytes received
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 3000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 3000k /dev/urandom > /tmp/tmp20ry_80f
Data Size: 3000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp20ry_80f  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 90 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.329511	0.000000	0.329568		7.895535	0		200		3072000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 700k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 700k /dev/urandom > /tmp/tmpu5t6k6vf
Data Size: 700k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpu5t6k6vf  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318921	0.000000	0.318976		2.735111	0		200		716800		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 22000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 22000k /dev/urandom > /tmp/tmpzxouzqew
Data Size: 22000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpzxouzqew  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 660 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319143	0.000000	0.319201		44.853447	0		200		22528000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 24000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 24000k /dev/urandom > /tmp/tmpxdfswp6r
Data Size: 24000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpxdfswp6r  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 720 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324381	0.000000	0.324438		53.996174	0		200		24576000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 26000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 26000k /dev/urandom > /tmp/tmpqomww147
Data Size: 26000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpqomww147  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 780 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318980	0.000000	0.319037		54.410626	0		200		26624000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 28000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 28000k /dev/urandom > /tmp/tmp_a8rm302
Data Size: 28000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp_a8rm302  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 840 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318963	0.000000	0.319019		61.498957	0		200		28672000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 32000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 32000k /dev/urandom > /tmp/tmptelrxgep
Data Size: 32000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmptelrxgep  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 960 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319351	0.000000	0.319407		66.180296	0		200		32768000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 34000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 34000k /dev/urandom > /tmp/tmprl4whf_j
Data Size: 34000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmprl4whf_j  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 1020 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319375	0.000000	0.319430		68.669098	0		200		34816000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 36000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 36000k /dev/urandom > /tmp/tmpqrve3p8z
Data Size: 36000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpqrve3p8z  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 1080 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324519	0.000000	0.324575		77.578237	0		200		36864000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 38000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 38000k /dev/urandom > /tmp/tmppeo2zbm3
Data Size: 38000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmppeo2zbm3  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 1140 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318884	0.000000	0.318939		77.952597	0		200		38912000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 11000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 11000k /dev/urandom > /tmp/tmpxhdych3d
Data Size: 11000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpxhdych3d  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 330 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330244	0.000000	0.330299		25.042599	0		200		11264000		18	
0.158289	0.000000	0.158345		24.137326	0		200		11264000		18	
0.158277	0.000000	0.158336		23.594264	0		200		11264000		18	
0.158313	0.000000	0.158370		22.128229	0		200		11264000		18	
0.158475	0.000000	0.158533		27.355008	0		200		11264000		18	
0.164037	0.000000	0.164093		22.173072	0		200		11264000		18	
0.158576	0.000000	0.158633		329.999012	28		000		11264000		0	Operation timed out after 329999 milliseconds with 0 bytes received
0.158410	0.000000	0.158466		25.336904	0		200		11264000		18	
0.164106	0.000000	0.164163		24.042820	0		200		11264000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 12000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 12000k /dev/urandom > /tmp/tmpvps_7z6_
Data Size: 12000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpvps_7z6_  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 360 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.325412	0.000000	0.325467		26.793529	0		200		12288000		18	
0.158267	0.000000	0.158323		26.396795	0		200		12288000		18	
0.164097	0.000000	0.164152		26.044505	0		200		12288000		18	
0.158264	0.000000	0.158320		24.214435	0		200		12288000		18	
0.164010	0.000000	0.164062		28.872634	0		200		12288000		18	
0.163995	0.000000	0.164045		27.965869	0		200		12288000		18	
0.158152	0.000000	0.158205		25.789612	0		200		12288000		18	
0.158161	0.000000	0.158214		23.199254	0		200		12288000		18	
0.158250	0.000000	0.158304		28.101995	0		200		12288000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 13000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 13000k /dev/urandom > /tmp/tmpro_2kcmg
Data Size: 13000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpro_2kcmg  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 390 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.326387	0.000000	0.326443		29.298640	0		200		13312000		18	
0.164084	0.000000	0.164142		26.485114	0		200		13312000		18	
0.158294	0.000000	0.158350		26.540950	0		200		13312000		18	
0.164048	0.000000	0.164106		27.928455	0		200		13312000		18	
0.158297	0.000000	0.158353		28.935911	0		200		13312000		18	
0.164141	0.000000	0.164199		27.504652	0		200		13312000		18	
0.163995	0.000000	0.164052		28.433973	0		200		13312000		18	
0.158285	0.000000	0.158342		29.910046	0		200		13312000		18	
0.164086	0.000000	0.164142		31.098635	0		200		13312000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 14000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 14000k /dev/urandom > /tmp/tmp_uwb_hth
Data Size: 14000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp_uwb_hth  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 420 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.328444	0.000000	0.328502		28.832081	0		200		14336000		18	
0.164122	0.000000	0.164180		27.646956	0		200		14336000		18	
0.158313	0.000000	0.158368		27.939057	0		200		14336000		18	
0.164126	0.000000	0.164185		30.338343	0		200		14336000		18	
0.158556	0.000000	0.158614		32.847068	0		200		14336000		18	
0.158306	0.000000	0.158359		27.907571	0		200		14336000		18	
0.164069	0.000000	0.164126		30.566567	0		200		14336000		18	
0.158304	0.000000	0.158374		30.415562	0		200		14336000		18	
0.158241	0.000000	0.158296		27.641623	0		200		14336000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 15000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 15000k /dev/urandom > /tmp/tmpeeop11uj
Data Size: 15000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpeeop11uj  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 450 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318913	0.000000	0.318969		32.750291	0		200		15360000		18	
0.164089	0.000000	0.164146		33.148754	0		200		15360000		18	
0.158362	0.000000	0.158417		29.396263	0		200		15360000		18	
0.164109	0.000000	0.164165		34.568462	0		200		15360000		18	
0.158257	0.000000	0.158311		449.999014	28		000		1044647		0	Operation timed out after 449999 milliseconds with 0 bytes received
0.164235	0.000000	0.164292		29.276446	0		200		15360000		18	
0.545629	0.000000	0.545688		32.911878	0		200		15360000		18	
0.158296	0.000000	0.158350		31.381336	0		200		15360000		18	
0.158307	0.000000	0.158363		30.556164	0		200		15360000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 16000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 16000k /dev/urandom > /tmp/tmp9di7ilu2
Data Size: 16000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp9di7ilu2  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 480 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319052	0.000000	0.319109		35.132785	0		200		16384000		18	
0.158424	0.000000	0.158498		36.060857	0		200		16384000		18	
0.158216	0.000000	0.158268		31.266469	0		200		16384000		18	
0.158307	0.000000	0.158363		31.084780	0		200		16384000		18	
0.164084	0.000000	0.164139		33.812700	0		200		16384000		18	
0.158493	0.000000	0.158548		32.223235	0		200		16384000		18	
0.164079	0.000000	0.164136		33.781181	0		200		16384000		18	
0.158261	0.000000	0.158315		36.250048	0		200		16384000		18	
0.164151	0.000000	0.164208		34.801287	0		200		16384000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 17000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 17000k /dev/urandom > /tmp/tmpueyzhrq7
Data Size: 17000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpueyzhrq7  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 510 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319067	0.000000	0.319121		37.511359	0		200		17408000		18	
0.158509	0.000000	0.158563		36.017701	0		200		17408000		18	
0.158316	0.000000	0.158372		34.694705	0		200		17408000		18	
0.158295	0.000000	0.158349		35.459007	0		200		17408000		18	
0.158541	0.000000	0.158599		34.824698	0		200		17408000		18	
0.158489	0.000000	0.158542		36.347385	0		200		17408000		18	
0.158404	0.000000	0.158459		34.064337	0		200		17408000		18	
0.158265	0.000000	0.158320		38.616100	0		200		17408000		18	
0.158289	0.000000	0.158344		34.532056	0		200		17408000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 18000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 18000k /dev/urandom > /tmp/tmpf6j_cf89
Data Size: 18000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpf6j_cf89  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 540 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319201	0.000000	0.319258		36.792876	0		200		18432000		18	
0.158295	0.000000	0.158350		35.400389	0		200		18432000		18	
0.158302	0.000000	0.158358		36.267466	0		200		18432000		18	
0.164020	0.000000	0.164074		36.632985	0		200		18432000		18	
0.158276	0.000000	0.158333		39.651728	0		200		18432000		18	
0.164173	0.000000	0.164230		37.924361	0		200		18432000		18	
0.158299	0.000000	0.158355		36.771705	0		200		18432000		18	
0.158248	0.000000	0.158311		41.835958	0		200		18432000		18	
0.158250	0.000000	0.158305		37.741999	0		200		18432000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 19000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 19000k /dev/urandom > /tmp/tmp8gw832wd
Data Size: 19000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp8gw832wd  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 570 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.325023	0.000000	0.325079		38.364324	0		200		19456000		18	
0.158220	0.000000	0.158270		39.728260	0		200		19456000		18	
0.164085	0.000000	0.164139		41.190044	0		200		19456000		18	
0.158517	0.000000	0.158572		42.916734	0		200		19456000		18	
0.164182	0.000000	0.164237		39.565018	0		200		19456000		18	
0.158534	0.000000	0.158586		40.936526	0		200		19456000		18	
0.158482	0.000000	0.158535		39.492919	0		200		19456000		18	
0.158299	0.000000	0.158356		40.528757	0		200		19456000		18	
0.164039	0.000000	0.164093		37.899759	0		200		19456000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 20000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 20000k /dev/urandom > /tmp/tmpi0wixrr4
Data Size: 20000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpi0wixrr4  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 600 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319097	0.000000	0.319151		43.125458	0		200		20480000		18	
0.158252	0.000000	0.158306		44.045551	0		200		20480000		18	
0.164156	0.000000	0.164211		46.379659	0		200		20480000		18	
0.158505	0.000000	0.158560		39.436116	0		200		20480000		18	
0.158383	0.000000	0.158439		599.999014	28		000		20480000		0	Operation timed out after 599999 milliseconds with 0 bytes received
0.158387	0.000000	0.158443		38.847816	0		200		20480000		18	
0.158156	0.000000	0.158209		41.231347	0		200		20480000		18	
0.158198	0.000000	0.158252		44.072357	0		200		20480000		18	
0.158270	0.000000	0.158324		41.248235	0		200		20480000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 22000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 22000k /dev/urandom > /tmp/tmplmybtukq
Data Size: 22000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmplmybtukq  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 660 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324532	0.000000	0.324592		47.509535	0		200		22528000		18	
0.164005	0.000000	0.164060		50.705817	0		200		22528000		18	
0.158269	0.000000	0.158330		46.546030	0		200		22528000		18	
0.158262	0.000000	0.158319		44.218218	0		200		22528000		18	
0.158334	0.000000	0.158391		46.277146	0		200		22528000		18	
0.158290	0.000000	0.158347		49.671516	0		200		22528000		18	
0.158276	0.000000	0.158331		44.037415	0		200		22528000		18	
0.158518	0.000000	0.158575		44.975873	0		200		22528000		18	
0.164129	0.000000	0.164186		48.004740	0		200		22528000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 24000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 24000k /dev/urandom > /tmp/tmp7fm1xnm6
Data Size: 24000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp7fm1xnm6  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 720 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324350	0.000000	0.324405		50.150944	0		200		24576000		18	
0.158336	0.000000	0.158392		47.565675	0		200		24576000		18	
0.158253	0.000000	0.158308		52.646557	0		200		24576000		18	
0.158293	0.000000	0.158351		49.920465	0		200		24576000		18	
0.158336	0.000000	0.158392		49.013672	0		200		24576000		18	
0.158337	0.000000	0.158395		49.708358	0		200		24576000		18	
0.542125	0.000000	0.542183		50.327955	0		200		24576000		18	
0.158291	0.000000	0.158346		50.385978	0		200		24576000		18	
0.158530	0.000000	0.158586		50.613353	0		200		24576000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 26000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 26000k /dev/urandom > /tmp/tmpa__h3tt_
Data Size: 26000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpa__h3tt_  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 780 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324964	0.000000	0.325019		56.285616	0		200		26624000		18	
0.158536	0.000000	0.158590		51.803571	0		200		26624000		18	
0.158249	0.000000	0.158305		57.875852	0		200		26624000		18	
0.158246	0.000000	0.158302		52.263140	0		200		26624000		18	
0.158230	0.000000	0.158283		53.279437	0		200		26624000		18	
0.158309	0.000000	0.158365		54.690156	0		200		26624000		18	
0.158233	0.000000	0.158286		53.872576	0		200		26624000		18	
0.158265	0.000000	0.158321		54.935927	0		200		26624000		18	
0.158303	0.000000	0.158359		56.803934	0		200		26624000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 28000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 28000k /dev/urandom > /tmp/tmpy73x3aeu
Data Size: 28000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpy73x3aeu  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 840 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.325079	0.000000	0.325136		59.406384	0		200		28672000		18	
0.158305	0.000000	0.158360		61.190456	0		200		28672000		18	
0.158285	0.000000	0.158342		56.150595	0		200		28672000		18	
0.158306	0.000000	0.158361		58.247391	0		200		28672000		18	
0.158229	0.000000	0.158289		59.722313	0		200		28672000		18	
0.164043	0.000000	0.164098		58.421032	0		200		28672000		18	
0.158497	0.000000	0.158552		61.325297	0		200		28672000		18	
0.158463	0.000000	0.158518		56.705065	0		200		28672000		18	
0.158248	0.000000	0.158302		55.835677	0		200		28672000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 30000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 9 ',)
head -c 30000k /dev/urandom > /tmp/tmpgvljlbm0
Data Size: 30000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpgvljlbm0  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 900 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.325079	0.000000	0.325135		64.842381	0		200		30720000		18	
0.158266	0.000000	0.158323		66.209632	0		200		30720000		18	
0.158307	0.000000	0.158364		60.900981	0		200		30720000		18	
0.158258	0.000000	0.158314		61.190099	0		200		30720000		18	
0.158277	0.000000	0.158333		61.869640	0		200		30720000		18	
0.158306	0.000000	0.158361		63.711920	0		200		30720000		18	
0.158313	0.000000	0.158369		66.786102	0		200		30720000		18	
0.164050	0.000000	0.164105		62.156256	0		200		30720000		18	
0.164184	0.000000	0.164242		61.037879	0		200		30720000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 11000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 11000k /dev/urandom > /tmp/tmpmglq2gqw
Data Size: 11000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpmglq2gqw  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 330 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318944	0.000000	0.319001		23.508464	0		200		11264000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 15000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 15000k /dev/urandom > /tmp/tmpztc96_sg
Data Size: 15000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpztc96_sg  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 450 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.318964	0.000000	0.319020		32.324116	0		200		15360000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
rm: cannot remove 'r1.log': No such file or directory
rm: cannot remove 'h2.log': No such file or directory
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) (10.00Mbit 1ms delay 1.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 20000k --http 3 --stdout /dev/null --stderr /dev/null -cc cubic --loss 1 --log-level error -t 1 ',)
head -c 20000k /dev/urandom > /tmp/tmpkljnr8l3
Data Size: 20000k
HTTP: --http3
RUST_LOG=error sidecurl --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpkljnr8l3  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 600 -o /dev/null 2>>/dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319135	0.000000	0.319195		43.773658	0		200		20480000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
